blog.html

```diff
@@ -43,8 +43,8 @@ <h2 class="bullet">
                     Up until now, only the default compression level was used by Apache Kafka. From this version on, a configuration mechanism to specify compression level is included. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-390%3A+Support+Compression+Level">KIP-390</a> for more details.
                 </p>
                 <p>
-                    In the last release, 3.7, <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol">KIP-848 The Next Generation of the Consumer Rebalance Protocol</a> was made available as early access. This version includes numerous bug
-                    fixes and the community is encouraged to test and provide feedback. <a href="https://cwiki.apache.org/confluence/display/KAFKA/The+Next+Generation+of+the+Consumer+Rebalance+Protocol+%28KIP-848%29+-+Early+Access+Release+Notes">See the early access release notes for more information.</a>
+                    <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-848%3A+The+Next+Generation+of+the+Consumer+Rebalance+Protocol">KIP-848 The Next Generation of the Consumer Rebalance Protocol</a> is available as preview in 3.8. This version includes numerous bug
```

Comment:
Do we have the concept of `preview` in Kafka?
I know we had several discussions in previous PRs regarding "early access" and "production ready" not being really defined (https://github.com/apache/kafka-site/pull/614#discussion_r1686561076 for example)

Reply:
I think that we don't have any official graduation terminology for new features in Apache Kafka. It may be worth to discuss it on the mailing list to determine them. That being said, I think that we used Preview in the past too.

From a previous blog post:

> KIP-796 is a long-term project that will be extended with new query types in future releases. As of Apache Kafka 3.2.0, IQv2 is in preview. The public documentation site has not been updated, and the interfaces of IQv2 are marked as @Evolving (meaning that they may break compatibility in minor releases without a deprecation period if preview users find significant flaws in the current API). A future release will remove the @Evolving annotation and designate IQv2 as stable.

I would like to stress in this release that KIP-848 has made significant progress since the last release from a quality and stability point of view. Our early access was really early from this point of view.

Reply:
Ah this is from the "old" blog posts. From 3.5.0 onwards I couldn't find any reference to "Preview".
I know what you mean with wanting to flag that this KIP moved "upwards" in terms of production readiness. Let's accept this here and I'll open a discuss thread in the mailing list to clarify our feature graduation steps.

Reply:
Thanks!